.yarn/versions/4aa0961e.yml

```
@@ -0,0 +1,34 @@
+releases:
+  "@yarnpkg/core": patch
```

```suggestion
  "@yarnpkg/core": minor
```
A new feature was added to the core.